Bottom: 65b0a986e9d5662639c6348e9366b6a97f84f628
Top:    7f027129a6843723f7b385c217f35c825c082c3c
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 13:38:15 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/lib/master.c b/lib/master.c
index 00c8a66..37046a3 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1389,7 +1389,7 @@ int master_show_mounts(struct master *master, const char *maps)
 		struct autofs_point *ap;
 		time_t now = time(NULL);
 		unsigned int count = 0;
-		char *match_name = NULL;
+		char *map_name = NULL;
 		int i;
 
 		this = list_entry(p, struct master_mapent, list);
@@ -1400,8 +1400,8 @@ int master_show_mounts(struct master *master, const char *maps)
 		printf("\nMount point: %s\n", ap->path);
 
 		if (maps) {
-			match_name = match_map_name(ap->path, maps);
-			if (!match_name) {
+			map_name = match_map_name(ap->path, maps);
+			if (!map_name) {
 				printf("\n");
 				continue;
 			}
